site/plugins.json

```
@@ -245,7 +245,17 @@
     "name": "Inline critical CSS",
     "package": "netlify-plugin-inline-critical-css",
     "repo": "https://github.com/Tom-Bonnike/netlify-plugin-inline-critical-css",
-    "version": "1.2.0"
+    "version": "2.0.0",
+    "compatibility": [
+      {
+        "version": "2.0.0",
+        "migrationGuide": "https://github.com/Tom-Bonnike/netlify-plugin-inline-critical-css/blob/master/CHANGELOG.md#200"
+      },
+      {
+        "version": "1.2.0",
+        "nodeVersion": "TBD"
```

I’m unsure what I should put here?! v2.0.0 contains a [`critical`](https://github.com/addyosmani/critical) major version update that drops support for Node v11 and v13. 😕

---

```suggestion
        "nodeVersion": "^11 || ^13"
```

I believe this [is the change](https://github.com/addyosmani/critical/commit/6b3e957ee2081dccb7113f23f4d648142687ef53#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R26), is this correct?

Then `1.2.0` should be used instead of `2.0.0` for sites with Node 11 or 13. To be honest, I believe this would be quite unlikely for some code to work on Node 10, 12 and 14, but not 11 or 13, so we might be on the safe side here.

---

> I believe this is the change, is this correct?

Yes.

> If so, 1.2.0 should be used instead of 2.0.0 for sites with Node 11 or 13.

Ahh, took me a while but yes, that should do it. Thanks!

The anyone on Node 10, 12 or 14 and up can use 2.0.0.

> Please note that sites currently using 1.2.0 will keep doing so until the user has explicitly upgraded the plugin in the UI (if the plugin was installed in the UI).

Understood! 👍